distribution/docs/src/main/resources/content/_running/starting-intro.adoc

```diff
@@ -177,7 +177,7 @@ RUN_AS_USER=<${branding-lowercase}-user>
 +
 . (Windows users skip to next step) (All *NIX) Edit `${home_directory}/bin/${branding-lowercase}-service`. Add LimitNOFILE to the [Service] section.
 +
-.${home_directory}/bin/${branding-lowercase}-service
+.<${branding}_HOME>/bin/${branding-lowercase}.service
```

Comment:
Why are we not using the home_directory variable here?

Reply:
I think this might have been an unintentional change from rebasing this changeset.